webapp/originalDataPoint/originalDataPointState.js

```diff
@@ -0,0 +1,15 @@
+import * as R from 'ramda'
+
+export const stateKey = 'originalDataPoint'
```

Comment:
should we call this nationalDataPoint? In the future we'll refactor the components and db tables. what do you think?